backend/src/test/java/heartbeat/service/report/PipelineServiceTest.java

```diff
@@ -58,18 +57,11 @@ public class PipelineServiceTest {
 	@Captor
 	ArgumentCaptor<Map<String, String>> roadMapArgumentCaptor;
 
-	private String token;
-
-	private String startTime;
+	public String token = "startTime";
 
-	private String endTime;
+	public String startTime = "endTime";
 
-	@BeforeEach
-	public void setup() {
-		startTime = "startTime";
-		endTime = "endTime";
-		token = "token";
-	}
+	public String endTime = "token";
```

Comment:
give the variable a meaningful value.
make code readable
@yulongcai

Reply:
done, pair with shiqi, on her pr